pkgs/applications/science/biology/freebayes/default.nix

```
@@ -2,14 +2,14 @@
 
 stdenv.mkDerivation rec {
   pname = "freebayes";
-  version = "2017-08-23";
+  version = "1.3.1";
 
   src = fetchFromGitHub {
     name = "freebayes-${version}-src";
     owner  = "ekg";
     repo   = "freebayes";
-    rev    = "8d2b3a060da473e1f4f89be04edfce5cba63f1d3";
-    sha256 = "0yyrgk2639lz1yvg4jf0ccahnkic31dy77q05pb3i28rjf37v45z";
+    rev    = "4cde5d32ff745018e0b8ec31e81a71e8bc9ecc6f";
```

allow for automatic bumps
```suggestion
    rev    = "v${version}";
```

---

I have no problem with this change, but how does automatic updating work?

---

there's a bot, @r-ryantm that can understand nix expressions enough to attempt to a version bump based upon the latest version in repology, and then update the sha.